### PR TITLE
[Spec] Added specs for module block & aptos_account & state_storage

### PR DESCRIPTION
**Pull request by 0xOutOfGas**
### Description
This is a part of spec work for Aptos Framework from MoveBit, we updated below files/modules in this PR:
> Added specs for module block & aptos_account & state_storage.

### Test Plan

> aptos move prove --package-dir ./aptos-framework

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/5478)

### CODE REVIEW

File: aptos-move/framework/aptos-framework/sources/aptos_account.spec.move [hunk 1]
```diff
@@ -0,0 +1,48 @@
+spec aptos_framework::aptos_account {
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
```
[the remark this answers is not in the document]
Comment (0xOutOfGas) [reply aptos-move/framework/aptos-framework/sources/aptos_account.spec.move:4]: This is to ensure all the functions are verified in this module.

File: aptos-move/framework/aptos-framework/sources/state_storage.spec.move [hunk 2]
```diff
@@ -1,20 +1,39 @@
 spec aptos_framework::state_storage {
     spec module {
         use aptos_std::chain_status;
+        pragma verify = true;
+        pragma aborts_if_is_strict;
```
[the remark this answers is not in the document]
Comment (0xOutOfGas) [reply aptos-move/framework/aptos-framework/sources/state_storage.spec.move:5]: This is to ensure all the functions are verified in this module.

File: aptos-move/framework/aptos-framework/sources/block.spec.move [hunk 3]
```diff
@@ -41,4 +41,87 @@ spec aptos_framework::block {
...
+    }
+
+    /// The caller is @aptos_framework.
+    /// The new_epoch_interval must be greater than 0.
+    /// The BlockResource existed under the @aptos_framework.
+    spec update_epoch_interval_microsecs(
+        aptos_framework: &signer,
+        new_epoch_interval: u64,
+    ) {
+        include Update_epoch_interval_microsecs;
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/block.spec.move:85]: Not: Why capitalize here? And other places as well

File: aptos-move/framework/aptos-framework/sources/block.spec.move [hunk 4]
```diff
@@ -41,4 +41,87 @@ spec aptos_framework::block {
...
+        use std::signer;
+        aptos_framework: signer;
+        new_epoch_interval: u64;
+
+        let addr = signer::address_of(aptos_framework);
+
+        aborts_if addr != @aptos_framework;
+        aborts_if new_epoch_interval <= 0;
+        aborts_if !exists<BlockResource>(addr);
+        ensures exists<BlockResource>(addr);
```
Comment (junkil-park) [inline aptos-move/framework/aptos-framework/sources/block.spec.move:98]: What's the reason for this to be re-ensured?

File: aptos-move/framework/aptos-framework/sources/block.spec.move (hunk 3 above)
Comment (junkil-park) [reply aptos-move/framework/aptos-framework/sources/block.spec.move]: Yeah, a convention is that schema names are in camelcase without `_`. I see this is corrected in the newer commit pushed.

**Review by junkil-park — APPROVED** [review]
I left a few minor comments. Other than that, it looks good to me.

**Review by movekevin — APPROVED** [review]

**Pull request merged**